rpa-manager/modules/ROOT/pages/botmanagement-manage.adoc

```diff
@@ -57,15 +57,15 @@ For example, if you specify a downtime that starts at 1:30 AM and ends at 5:30 A
 
 Learn how to xref:processautomation-prepare-project-application.adoc[manage applications].
 
-. In the *Bots* view of the *Bot Management* module, click the *Edit* icon image:edit-icon.png[pen-to-square symbol,1.5%,1.5%] in the row of the bot to edit.
+. In the *Bots* view of the *Bot Management* module, click the *Edit* icon image:edit-icon.png[pen-to-paper symbol,1.5%,1.5%] in the row of the bot to edit.
```

Comment:
I learned recently that alt text with spaces or punctuation should be enclosed in quotes. Otherwise, it doesn't show up correctly.